driver/dostuff.py

```diff
@@ -40,11 +43,19 @@
                ]
 Engines, NumUpdated = builders.build(KnownEngines, not options.noupdate, options.force)
 
+Submitter = None
+if options.submitter == 'remote':
+    Submitter = submitter.RemoteSubmitter
+elif options.submitter == 'print':
+    Submitter = submitter.PrintSubmitter
+else:
+    raise Exception('unknown submitter!')
```

Comment:
Can you create getSubmitter(name) which has this logic.
Like:
https://github.com/bnjbvr/arewefastyet/blob/submitter/browser-driver/benchmarks_local.py#L75